assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -17,109 +15,225 @@ import {getAvailableParameters} from './selectors/parameter-data';
-                store.dispatch(setSelectedParameterCode(d.parameterCode));
+                store.dispatch(setSelectedParameterCode(parameter.parameterCode));
                 showDataIndicators(true, store);
                 store.dispatch(retrieveHydrographData(siteno, getInputsForRetrieval(store.getState())))
                     .then(() => {
                         showDataIndicators(false, store);
                     });
             }
-        })
-        .call(tr => {
-            const paramSelectCol = tr.append('td');
-            paramSelectCol.append('input')
-                .attr('id', d => `time-series-select-radio-button-${d.parameterCode}`)
-                .attr('type', 'radio')
-                .attr('name', 'param-select-radio-input')
-                .attr('class', 'usa-radio__input')
-                .attr('value', d => `${d.parameterCode}`)
-                .call(link(store, (inputElem, selectedParameterCode) => {
-                    inputElem.property('checked', d => d.parameterCode === selectedParameterCode ? true : null);
-                }, getSelectedParameterCode));
-            paramSelectCol.append('label')
-               .attr('class', 'usa-radio__label');
-            const paramCdCol = tr.append('th')
-                .attr('scope', 'row');
-            paramCdCol.append('span')
-                .text(d => d.description)
-                .each(function(d) {
-                    appendInfoTooltip(select(this), `Parameter code: ${d.parameterCode}`);
-                });
-            tr.append('td')
-                .style('white-space', 'nowrap')
-                .text(d => d.periodOfRecord ?
-                    `${d.periodOfRecord.begin_date} to ${d.periodOfRecord.end_date}` : '');
-            tr.append('td')
-                .append('div')
-                .attr('class', 'wateralert-link');
         });
+};
 
-    // WaterAlert does not support every parameter code, so lets take that into account when adding the links
-    table.selectAll('.wateralert-link').each(function(d) {
-        let selection = select(this);
+/*
+* Helper function that creates the top row of each parameter selection. This row is hidden except on narrow screens
+* and contains the period of record that appears above the parameter description.
+* @param {Object} container - The target element to append the row
+* @param {D3 selection} parameter - Contains details about the current parameter code
+*/
+const drawTopPeriodOfRecordRow = function(container, parameter) {
+    const gridRowInnerTopPeriodOfRecord = container.append('div')
+        .attr('class', 'grid-row-inner grid-row-period-of-record');
+    gridRowInnerTopPeriodOfRecord.append('div')
+        .attr('class', 'grid-row-period-of-record-text')
+        .text(`${parameter.periodOfRecord.begin_date} to ${parameter.periodOfRecord.end_date}`);
+    const TopPeriodOfRecordRowExpansionControlDiv = gridRowInnerTopPeriodOfRecord.append('div')
```

Comment:
.Shouldn't start variable name with uppercase

Reply:
Rats! I'm surprised the linter doesn't catch that.